src/bn_wallets.erl

```diff
@@ -176,10 +176,14 @@ handle_rpc(<<"wallet_pay">>, {Param}) ->
     Payee = ?jsonrpc_b58_to_bin(<<"payee">>, Param),
     Amount = ?jsonrpc_get_param(<<"bones">>, Param, undefined),
     Max = ?jsonrpc_get_param(<<"max">>, Param, false),
+    MemoBin = ?jsonrpc_get_param(<<"memo">>, Param, undefined),
```

Comment:
Are memos part of or required by the token/network transactions? If this hasn’t been an available field previously do we want to include it now?

Reply:
Deleted.